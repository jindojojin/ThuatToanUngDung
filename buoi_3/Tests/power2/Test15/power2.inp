70
12 42 5 16 43 54 33 4 3 17 7 18 5 1 5 3 3 19 5 111 5 3 3 11 7 111 234 5 567 5 3 3 1 5 1 5 3 3 7 4 5 5 3 4 3 5 2048 5 3 3 7 5 5 3 3 5 1024 5 3 4 3 7 512 1 789 5 3 12121 3 2
