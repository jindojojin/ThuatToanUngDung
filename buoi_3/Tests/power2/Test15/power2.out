1 1 1 1 2 4 4 4 4 16 512 1024 2048
